.github/workflows/tests.yaml

```diff
@@ -22,13 +20,6 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu", "macos", "windows"]
```

Comment:
```suggestion
        os: ["ubuntu-latest", "macos-latest", "macos-14", "windows-latest"]
```

Reply:
yay! yes this is what I was asking on your PR